### DISCUSSION TITLE
Account suspended and non-existent github support

### DISCUSSION THREAD

**Discussion by ghost** [community/community#128775] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Last week my @felipolis account was suspended. When I tried to log in I received the message "Account suspended. Access to your account has been suspended due to a violation of our Terms of Service."

I have no idea why this happened. I didn't receive any alert email or anything.

I depend on GitHub for two things: work and college.

My work repositories are stored there, as well as my college projects.

Given this, I beg you to reinstate my account again.

I had this problem last week and so far I haven't received a response.

My ticket is #2838644

**Comment by jayzgame99** [comment]
Me too! My Account got suspended and still no response from support team.

Ticket ID: **2839418**
Email: **boredguys.dev@gmail.com**

Git Team, please response! 

Thank you!

**Comment by BeyondEternalDEV** [comment]
Good luck getting a response, I opened ticket (#2684388) for the same reason since MARCH 19 and still no response.

**Reply by ghost** [reply]
This is very bizarre... There are people who depend on this for work.

**Reply by BeyondEternalDEV** [reply]
Yeah, I'm still waiting for an answer

**Comment by Mee-meet** [comment]
Subject: Request for Unsuspension of GitHub Account

Dear GitHub Support Team,

I hope this message finds you well. My name is Meet Mehta, and I am writing to request the unsuspension of my GitHub account. The details of my account are as follows:

Username: MeetInCode
Email: meets.studentlife@gmail.com
I recently discovered that my account has been suspended, and I am unsure of the reason for this action. I rely heavily on GitHub for my studies and projects, and losing access to my repositories has caused significant disruption.

If the suspension was due to a misunderstanding or an error on my part, I sincerely apologize and am willing to take any necessary corrective actions to ensure compliance with GitHub's terms of service.

I would greatly appreciate your assistance in reviewing my case and providing guidance on how to resolve this issue. Thank you for your understanding and support.

Looking forward to your prompt response.

Best regards,
Meet Mehta

**Comment by misterpip** [comment]
same problem here..
Ticket Number: 2834057

**Comment by queenofcorgis** [comment, marked as answer]
Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! In the future, please only open 1 ticket per incident.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
